backend/doc/decisions/5-use-yarn.md

```diff
@@ -0,0 +1,10 @@
+Decided to use yarn package manager instead of npm because of the [issue](https://github.com/yarnpkg/yarn/issues/5962)
```

Comment:
Reading this issue I get the impression that Yarn in fact has a bug, not npm. What issue were you having with npm?

I'd really prefer not to mix both yarn and npm into the project, so if we must use yarn here I'd prefer to use it across the board.

Reply:
There are many incarnations of this problem, such as [this](https://stackoverflow.com/questions/38058386/why-is-node-gyp-rebuild-failing-on-mac-osx-el-capitan) and [this](https://github.com/nodejs/node-gyp/issues/1779). Googling "npm [node-gyp|fsevents] mac os" shows them.

I spent maybe a couple of hours trying to work around this problem and just can't fix it (`npm install` was failing). I suggest sticking to yarn until somebody of the Climatescape's developers stumbles upon a similar problem that just can't be solved in `yarn`.

Reply:
Sounds good, we can worry about it later